rotkehlchen/api/v1/schemas.py

```diff
@@ -2318,3 +2318,7 @@ def validate_schema(  # pylint: disable=no-self-use
                 message='Can\'t delete the etherscan node',
                 field_name='identifier',
             )
+
+
+class DetectTokensSchema(AsyncQueryArgumentSchema):
```

Comment:
This is the exact same schema as `AddressbookAddressesSchema` uses. Abstract it into a common schema name: `OptionalAddressesListSchema` and then use it both in `AddressbookAddressesSchema` and in the one you are making here. Using it would probably simply mean inheriting, essentially like implementing an interface.

Reply:
Done